README.md

```diff
@@ -1,18 +1,49 @@
 # Azure Exporter for Scrapy
+[Scrapy feed export storage backend](https://doc.scrapy.org/en/latest/topics/feed-exports.html#storage-backends) for [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/).
 
-## Requirements:
-- Python 3.8
-- Pipenv
+## Installation
+```bash
+pip install git+https://github.com/scrapy-plugins/scrapy-feedexporter-azure-storage
+```
+## Usage
+* Add this storage backend to the [FEED_STORAGES](https://docs.scrapy.org/en/latest/topics/feed-exports.html#std-setting-FEED_STORAGES) Scrapy setting. For example:
+```python
+# settings.py
+FEED_STORAGES = {'azure': 'scrapy_azure_exporter.AzureFeedStorage'}
+```
+* Provide [Authentication](https://docs.microsoft.com/en-us/python/api/overview/azure/storage-blob-readme?view=azure-python) for Azure account via any of the following settings.
+  - `AZURE_CONNECTION_STRING` 
+  - `AZURE_ACCOUNT_URL_WITH_SAS_TOKEN`
+  - `AZURE_ACCOUNT_URL` & `AZURE_ACCOUNT_KEY` - If using this method, specify both of them.
+```python 
+For example,
+AZURE_ACCOUNT_URL = "https://<your-storage-account-name>.blob.core.windows.net/"
+AZURE_ACCOUNT_KEY = "Account key for the Azure account"
+```
```

Comment:
“For example” should be outside the code block, and you should make sure it and the code block are nested within the bullet point where they should (I am not 100% sure that my suggestion does).

```suggestion
  For example:
  ```python 
  AZURE_ACCOUNT_URL = "https://<your-storage-account-name>.blob.core.windows.net/"
  AZURE_ACCOUNT_KEY = "Account key for the Azure account"
  ```
```